.github/workflows/pr-labeler.yaml

```
@@ -0,0 +1,12 @@
+name: pr-labeler
+on:
+  - pull_request
```

I think you need `:`.
```suggestion
  - pull_request:
```

---

This will be outdated